src/registry_schemas/schemas/correction.json

```diff
@@ -116,6 +116,68 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "required": ["legalType"],
+            "properties": {
+              "legalType": {
+                "enum": [
+                  "BEN"
+                ]
+              }
+            }
+          },
+          "then": {
+            "required": [
+              "nameRequest",
+              "offices",
```

Comment:
I don't think 'nameRequest' and 'offices' should be required properties.  If you are not correcting name request then you need not pass that object at all. Same with offices. Party can be mandatory if the completing party is always mandatory